### PR TITLE
Update UserGuide for Doctor Command

### LINKED ISSUE #187: Update UG: Doctor Management Features

### PR DESCRIPTION
**Pull request by kimshitong**
Close #187

### CODE REVIEW

File: docs/UserGuide.md
Comment (mounilsankar) [inline docs/UserGuide.md:1]: this part on edit Doctor should have sample usage

File: docs/UserGuide.md [hunk 1]
```diff
@@ -367,6 +400,48 @@ Examples:
...
+* The index refers to the index number shown in the displayed doctor list.
+* The index **must be a positive integer** 1, 2, 3, …
+
+Examples:
+* `list_d` followed by `delete_tag_d 2 Orthopaedic` add specialisation the 2nd doctor in the doctor list.
+* `find_d Jonathan` followed by `delete_tag_d 1 Orthopaedic` add specialisation the 1st doctor in the results of the `find_d` command.
+
+### Editing a remark : `remark_d`
+
+Edits the remark of a doctor from the system.
```
Comment (yanghengtang) [inline docs/UserGuide.md:433]: Can talk about what a remark can be used for. In our case we talk about indicating the doctor's availability.

File: docs/UserGuide.md [hunk 2]
```diff
@@ -332,6 +350,21 @@ Format: `edit_d INDEX [ic\NRIC] [n\NAME]`
 Examples:
 *  `edit_d 1 n\Joe Ng ic\T0212385J` Edits the name and NRIC of the 1st doctor to be `Joe Ng` and `T0212385J` respectively.
 
+Sample Usage:
+1. Assuming you want to edit the 4th Doctor changing his phone from "T0212384J" to "T0212385J".
```
Comment (yanghengtang) [inline docs/UserGuide.md:354]: `Phone` change to `NRIC`

File: docs/UserGuide.md [hunk 3]
```diff
@@ -332,6 +350,21 @@ Format: `edit_d INDEX [ic\NRIC] [n\NAME]`
...
+```
+edit_d 4 ic\T0212385J
+```
+
+3. The result box will display the following message:
+```
+"Edited Doctor: Joe Ng Nric: T0212385J"
+```
+
+4. You have successfully edited the patient into the database.
```
Comment (yanghengtang) [inline docs/UserGuide.md:366]: `patient` change to `doctor`

**Review by yanghengtang — COMMENTED** [review]
Try to make the user guide sound more friendly

File: docs/UserGuide.md (hunk 1 above)
Comment (kimshitong) [reply docs/UserGuide.md]: done in new update! ty

**Review by derekjxtan — APPROVED** [review]
if there are any outstanding issues resolve in the next pr

**Pull request merged**